chore: generate capacitor 5 plugin docs (#159)
ionic-team/capacitor-docs: docs/apis/push-notifications.md

Check permission to receive push notifications.

> On Android 12 and below the status is always granted because you can always

Reviewer: ```suggestion
On Android 12 and earlier the status is always granted because no permission is needed
```

Request permission to receive push notifications.

> On Android 12 and below it doesn't prompt for permission because you can always

Reviewer: ```suggestion
On Android 12 and earlier the status is always granted because no permission is needed
```
